ui/razercfg

```diff
@@ -387,6 +414,7 @@ def usage():
 	print("-R|--getres                         Prints the resolutions")
 	print("-f|--freq [PROF:]FREQ               Changes the scan frequency")
 	print("-F|--getfreq                        Prints the frequencies")
+	print("-n|--mapping [PROF:]RES[xRES]       Changes the resolution of the currently used mapping")
```

Comment:
Wouldn't -m be more fitting? The existing arguments in the code are consistent, uppercase gets information, and lowercase sets information. Since this argument sets the resolution map, I think it should be lowercase m.